CONTRIBUTING.md

```diff
@@ -76,3 +76,61 @@ You should, of course, adapt this header to use the specific mechanism for comme
 ````
 
 when adding an XML file.
+
+## How to Contribute to an Existing Chart
+
+1. Fork this repository, develop and test your Chart changes. Remember to sign off your commits as described in the [Making your Changes](#making-your-changes) chapter.
+1. Ensure your Chart changes follow the [technical](#technical-requirements) and [documentation](#documentation-requirements) guidelines, described below.
+1. Submit a pull request.
+
+***NOTE***: In order to make testing and merging of PRs easier, please submit changes to multiple charts in separate PRs.
+
+### Technical Requirements
+
+* All Chart dependencies should also be submitted independently
+* Must pass the linter (`helm lint`)
+* Must successfully launch with default values (`helm install .`)
+    * All pods go to the running state (or NOTES.txt provides further instructions if a required value is missing e.g. [minecraft](https://github.com/helm/charts/blob/master/stable/minecraft/templates/NOTES.txt#L3))
+    * All services have at least one endpoint
+* Must include source GitHub repositories for images used in the Chart
+* Images should not have any major security vulnerabilities
+* Must be up-to-date with the latest stable Helm/Kubernetes features
+    * Use Deployments in favor of ReplicationControllers
+* Should follow Kubernetes best practices
+    * Include Health Checks wherever practical
+    * Allow configurable [resource requests and limits](http://kubernetes.io/docs/user-guide/compute-resources/#resource-requests-and-limits-of-pod-and-container)
+* Provide a method for data persistence (if applicable)
+* Support application upgrades
+* Allow customization of the application configuration
+* Provide a secure default configuration
+* Do not leverage alpha features of Kubernetes
+* Includes a [NOTES.txt](https://github.com/helm/helm/blob/master/docs/charts.md#chart-license-readme-and-notes) explaining how to use the application after install
+* Follows [best practices](https://github.com/helm/helm/tree/master/docs/chart_best_practices)
+  (especially for [labels](https://github.com/helm/helm/blob/master/docs/chart_best_practices/labels.md)
+  and [values](https://github.com/helm/helm/blob/master/docs/chart_best_practices/values.md))
+
+### Documentation Requirements
+
+* Must include an in-depth `README.md`, including:
+    * Short description of the Chart
+    * Any prerequisites or requirements
+    * Customization: explaining options in `values.yaml` and their defaults
```

Comment:
This item is tricky. I agree, that all configuration options has to be documented.
But I prefer to do the actual documentation in the `values.yaml` itself (see [Ditto config](https://github.com/eclipse/packages/blob/master/charts/ditto/values.yaml) for instance) and only reference to documentation in values.yaml from `README`

From our experience this has the benefit that documentation is close to where configuration is done (with all the resulting advantages).

What do you think?

Reply:
That makes a lot of sense to me :-) I'll change it accordingly ...